content/learn/workflow.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Sample Quire Workflow
+weight: 6600
+type: page
+abstract:  "An example of Getty Publication's workflow for Quire books"
+---
+
+Below, we outline the workflow Getty Publications's follows when creating [Quire books](https://www.getty.edu/publications/digital/digitalpubs.html). We understand that many organizations have smaller staffs working on publications, but we hope this documentation of our own internal process can provide insight and serve as a jumping off point for developing your organization's own procedure.
+
+## Project Roles
+
+Author(s) <br/>
+Lead project editor <br/>
+Digital team <br/>
+Copy editor<br/>
+Volume editor(s) <br/>
+Proofreader <br/>
+Production
+
+*Note: The lead project editor and copy editor have a basic understanding of Quire and GitHub, while the digital team is more advanced in their knowledge.*
+
+## Pre-transmittal
+
+-   The **authors** submit their manuscripts as Word files to the **lead project editor.**
+
+-   The document undergoes editorial review and peer review before being sent back to the **authors** for review.
+
+-   At this point the **lead project editor** meets with the **digital team** for a pre-transmittal meeting. Three things are covered:
+
+1. Review any special tagging that needs to be done to the files to ensure a clear conversion from Word document to [Markdown](/documentation/fundamentals/#markdown-basics) file. This helps the text-editing process run smoothly and efficiently.
+
+2. Identify special needs such as the use of videos, pop-ups, IIIF images, etc.
+
+3. Assess the content model of the book to identify unique attributes or places where the pattern of the book diverges from the norm. Quire is built to look for patterns so this is a critical step that will inform the data structuring of the publication and YAML formatting.
+
+-   The **lead project editor** gives the **copy editor** a memo outlining the tagging instructions. The **copy editor** enters the **authors'** changes and simultaneously “tags” the manuscript.
+
+<div class="box tip">
+
+- See our recommended [*Manuscript Prep*](/learn/manuscript-prep/) for information on how to prepare Word documents for a smooth conversion to Markdown.
+
+</div>
+
+## Transmittal
```

Comment:
I would combine this short section with the first pages section below and make it "Transmittal and First Pages". Transmittal is just the hand off, all the work that gets done after that is first pages.